lib/Platform/Intl/PlatformIntlApple.mm

```diff
@@ -68,17 +221,94 @@
   return canonicalizeLocaleList(runtime, locales);
 }
 
+llvh::Optional<std::u16string> localeListToLocaleString(
+    vm::Runtime *runtime,
+    const std::vector<std::u16string> &locales) {
+  // 3. Let requestedLocales be ? CanonicalizeLocaleList(locales).
+  vm::CallResult<std::vector<std::u16string>> requestedLocales =
+      canonicalizeLocaleList(runtime, locales);
+  if (LLVM_UNLIKELY(requestedLocales == llvh::ExecutionStatus::EXCEPTION)) {
+    //return llvh::ExecutionStatus::EXCEPTION;
```

Comment:
```suggestion
  if (LLVM_UNLIKELY(requestedLocales == vm::ExecutionStatus::EXCEPTION)) {
    return vm::ExecutionStatus::EXCEPTION;
```

Reply:
It's still inisisting that No viable conversion from returned value of type 'hermes::vm::ExecutionStatus' to function return type 'llvh::Optional<std::u16string>' (aka 'Optional<basic_string<char16_t>>')?

Reply:
See my other comment about changing the return type to `CallResult`, that will make the return type line up correctly.